hazelcast/src/main/java/com/hazelcast/map/impl/operation/WithForcedEviction.java

```diff
@@ -16,210 +16,23 @@
 
 package com.hazelcast.map.impl.operation;
 
-import com.hazelcast.config.MapConfig;
-import com.hazelcast.logging.ILogger;
-import com.hazelcast.map.impl.MapContainer;
-import com.hazelcast.map.impl.MapServiceContext;
-import com.hazelcast.map.impl.PartitionContainer;
-import com.hazelcast.map.impl.eviction.Evictor;
-import com.hazelcast.map.impl.recordstore.RecordStore;
-import com.hazelcast.memory.NativeOutOfMemoryError;
-import com.hazelcast.spi.impl.NodeEngine;
-import com.hazelcast.spi.impl.operationservice.BackupOperation;
-import com.hazelcast.spi.impl.operationservice.Operation;
-import com.hazelcast.spi.impl.operationservice.OperationService;
-import com.hazelcast.spi.properties.HazelcastProperties;
 import com.hazelcast.spi.properties.HazelcastProperty;
 
-import java.util.concurrent.ConcurrentMap;
-import java.util.logging.Level;
-
-import static com.hazelcast.config.EvictionPolicy.NONE;
-import static com.hazelcast.config.InMemoryFormat.NATIVE;
-import static com.hazelcast.internal.util.EmptyStatement.ignore;
-import static java.lang.String.format;
-
-/**
- * Includes retry logic when a map operation fails to put an entry
- * into {@code IMap} due to a {@link NativeOutOfMemoryError}.
- * <p>
- * If an {@code IMap} is evictable, naturally expected thing is, all put
- * operations should be successful. Because if there is no more space,
- * operation can be able to evict some entries and can put the new ones.
- * <p>
- * This abstract class forces the evictable record-stores on this partition
- * thread to be evicted in the event of a {@link NativeOutOfMemoryError}.
- * <p>
- * Used when {@link com.hazelcast.config.InMemoryFormat InMemoryFormat}
- * is {@link com.hazelcast.config.InMemoryFormat#NATIVE NATIVE}.
- */
-public final class WithForcedEviction {
-
-    public static final int DEFAULT_FORCED_EVICTION_RETRY_COUNT = 5;
-    public static final String PROP_FORCED_EVICTION_RETRY_COUNT
-            = "hazelcast.internal.forced.eviction.retry.count";
-    public static final HazelcastProperty FORCED_EVICTION_RETRY_COUNT
-            = new HazelcastProperty(PROP_FORCED_EVICTION_RETRY_COUNT,
-            DEFAULT_FORCED_EVICTION_RETRY_COUNT);
+final class WithForcedEviction {
```

Comment:
can you also make all access modifiers in this class, including class itself, `public`?